topics/github-desktop/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: GitHub
+display_name: GitHub Desktop
+github_url: https://github.com/desktop/desktop
+logo: github-desktop.png
+related: git, version-control
+released: September 2017
+short_description: GitHub Desktop is a graphical user interface for the Git version control system.
+topic: github-desktop
+url: https://desktop.github.com/
+wikipedia_url: https://en.wikipedia.org/wiki/GitHub_Desktop
```

Comment:
This page redirects to the `GitHub` page, so I don't think it's particularly useful here:

```suggestion
```